File: 3st/Juram/programmer/홀수 vs 짝수.js
function solution(num_list) {
    var oddSum = 0;
    var evenSum = 0;
    for(let i=0; i<num_list.length; i+=2){
        evenSum += num_list[i];
    }
    for(let i=1; i<num_list.length; i+=2){
        oddSum += num_list[i];
    }
    if(oddSum > evenSum) return oddSum;
    else if(oddSum < evenSum) return evenSum;
    else return evenSum;
}

File: 5st/gyuri/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
         int oddSum = 0;
        int evenSum = 0;
        
        for(int i=0; i<num_list.length; i++){
            if(i % 2 == 0){
                oddSum += num_list[i];
            }else {
                evenSum += num_list[i];
            }
        }
            
        return Math.max(oddSum, evenSum);
    }
}